checklists/apim_checklist.en.json

```diff
@@ -144,13 +164,53 @@
         {
             "category": "Management",
             "subcategory": "Business continuity and disaster recovery",
-            "text": "Deploy at least two scale units spread over two availability zones per region for best availability and performance",
+            "text": "Leverage multi-region deployment of API gateways (available only in premium tiers). APIM routes the requests to a regional gateway based on the lowest latency. You could use your own Traffic Manager with custom routing rules.",
+            "waf": "Reliability",
+            "service": "APIM",
+            "guid": "8d9b953c-cc0d-42c6-8e4f-24ef372e63c5",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/azure/api-management/"
+        },
+        {
+            "category": "Management",
+            "subcategory": "Request Routing",
+            "text": "In multi-region model, use Policies to route the requests to regional backends based on availability or latency.",
+            "waf": "Reliability",
+            "service": "APIM",
+            "guid": "1b8d68a4-66cd-44d5-ba94-3ee94440e8d6",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/azure/api-management/"
+        },
+        {
+            "category": "Management",
+            "subcategory": "Business continuity and disaster recovery",
+            "text": "Deploy at least two scale units per region for best availability and performance",
```

Comment:
The text should be "Deploy at least one unit in two or more availability zones for an increased SLA of 99.99%"

Reply:
Done!